### ISSUE TITLE
Translate Notebooks to Spanish

### ISSUE THREAD

**Issue by bnord** [lsst-epo/citizen-science-notebooks#28]
Wayy in the future: Translation for projects and or really trying to engage Chilean audience?

**Comment by clareh** [comment]
Yes. We definitely have the "mechanism" to translate our documentation into Spanish already. Zooniverse can apparently present in Spanish (functionality exists but isn't live), the outstanding part will be the actual notebooks
